icons/folder-queue.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?> <!-- Created with Inkscape (http://www.inkscape.org/) -->
```

Comment:
hey :) I think you checked in the wrong folder icon for the collapsed folder, right? It's also opened 🙈

Reply:
Geez, haven't seen it ... It's probably because i removed break lines to short svg, and may have taken the wrong one 😳

Now it's fixed!

Reply:
Thank you :)